packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -148,18 +163,28 @@ class AnalyzeCommand extends FlutterCommand {
 
     if (argResults['current-package']) {
       // ./lib/main.dart
-      String mainPath = 'lib/main.dart';
+      String mainPath = path.join('lib', 'main.dart');
```

Comment:
I'm not convinced this is needed. Are we sure `/` doesn't work on windows?
According to https://pub.dartlang.org/packages/path , it looks like it does ("both "/" and "\" are valid path separators, not just "\"").